man/os-release.xml

```diff
@@ -325,6 +325,31 @@
 
           <xi:include href="version-info.xml" xpointer="v249"/></listitem>
         </varlistentry>
+
+        <varlistentry>
+          <term><varname>RELEASE_TYPE=</varname></term>
+
+          <listitem><para>A lower-case string (no spaces or other characters outside of 0-9, a-z, ".",
+          "_", and "-"), describing what kind of release this version of the OS is. Known values follow:
+          <literal>stable</literal> is for normal releases of the system, suitable for production use.
+          Generally, stable releases become end-of-life soon after the next major stable release is out.
+          Examples include Fedora 40, Debian 12 Bookworm, Ubuntu 23.10, OpenSUSE Tumbleweed, and Arch Linux.
+          <literal>lts</literal> is for long term support releases of the system, suitable for production
+          use and supported for an extended period of time. Generally, LTS releases continue to recieve
+          support even if newer major releases of the distribution are available. Examples include Ubuntu
+          24.04 and RHEL 9.4.
+          <literal>pre-release</literal> is for unstable versions of the system, unsuitable for production
+          use, such as alpha, beta, or rolling unstable releases. Examples include Fedora Rawhide, Debian
+          Sid, Fedora 40 Beta, and GNOME OS Nightly.
```

Comment:
Sid is not a good example as it is in practice a rolling release, Debian Testing would be a good example